odor/src/main/scala/PostgresClient.scala

```diff
@@ -58,7 +58,7 @@ class PostgresConnectionPool(poolConfig: PgPoolConfig[PgClient], val logQueryTim
   def useConnection[R](code: PostgresClient => Future[R]): Future[R] = async {
     val pgClient = new PostgresClient(this)
 
-    val codeResult = await(code(pgClient).attempt)
+    val codeResult = Either.catchNonFatal(await(code(pgClient)))
```

Comment:
await cannot be used in lambdas. In this case you have to use a `try`/`catch` block which returns `Left` or `Right`.